src/App.test.js

```diff
@@ -50,5 +83,10 @@ describe('App component', () => {
     const userEmail = getByText(response.data.user.Email);
 
     expect(userEmail).toBeDefined();
+    expect(axios.get).toHaveBeenCalledTimes(1);
+    expect(axios.get).toHaveBeenCalledWith(process.env.REACT_APP_API_URL + '/Reports/Reports/GetUserData', {
```

Comment:
Maybe this `process.env.REACT_APP_API_URL + '/Reports/Reports/GetUserData' ` can be defined in a variable to reuse.